dashboard/app/controllers/levels_controller.rb

```diff
@@ -203,6 +204,10 @@ def embed_level
     render 'levels/show'
   end
 
+  def set_applab_user_id
+    @applab_user_id = applab_user_id
```

Comment:
applab_user_id seems to be defined in LevelsHelper already, you can just call the method